NYPL/cantaloupe
# Feature/dr 2873 short circuit rights lookup

## Ticket:

Short circuit rights lookup for homepage images. [DR-2873](https://jira.nypl.org/browse/DR-2873).

## Things Done

- Added a bypass for image ids used in the homepage to pull rights from pre-cooked rights information stored in a hash.
- Added unit tests

## How to Test

- Locally, run unit tests in jruby: `jruby test/delegates_test.rb`
- Run container locally and test image responses for image_ids from lanes data found here: https://github.com/NYPL/digital-collections/blob/main/src/data/lanes.ts
- Go to the logs in AWS console and search for any sign of the homepage image_ids. You should see no activity in repo api for those image_ids. 

## Concerns

- This method should eventually be removed once Repo API responses are improved. This is not meant to be a longterm solution.
